inference/nlp/realtime/llm/bloom_176b/djl_deepspeed_deploy.ipynb

```diff
@@ -0,0 +1,613 @@
+{
```

Comment:
<p>Perhaps it would be better to wrap this optional model download in a helper function, then comment it out the call to the function. That way it doesn't run on CI or when a customer just runs the whole notebook first, without actually reading through it first.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3642/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
<p>sure made the changes and added explicit conditions which need to be changed before this section will run</p>